src/coreclr/vm/stublink.h

```diff
@@ -464,6 +465,7 @@ class Stub
         LOADER_HEAP_BIT         = 0x20000000,
         INSTANTIATING_STUB_BIT  = 0x10000000,
         UNWIND_INFO_BIT         = 0x08000000,
+        THUNK_BIT               = 0x04000000,
 
         CODEBYTES_MASK          = UNWIND_INFO_BIT - 1,
```

Comment:
Should `CODEBYTES_MASK` be updated to account for the new flag?

Reply:
Ugh. Yes. Thanks for catching this. Thankfully it doesn't matter at present, but it needs to be updated.

Reply:
https://github.com/dotnet/runtime/pull/88830